contrib/sparse-checkout/update-grpc.sh

```diff
@@ -6,12 +6,12 @@ FILES_TO_CHECKOUT=$(git rev-parse --git-dir)/info/sparse-checkout
 echo '/*' > $FILES_TO_CHECKOUT
 echo '!/test/*' >> $FILES_TO_CHECKOUT
 echo '/test/build/*' >> $FILES_TO_CHECKOUT
+echo '/test/core/tsi/alts/fake_handshaker/*' >> $FILES_TO_CHECKOUT
```

Comment:
Why do we need these files (both test/build and test/core)? Are they needed for the build to succeed?

Reply:
Yes, gRPC moved some files around with the result that we now need to check them out (even if it looks silly).